konfera/models/ticket_type.py

```diff
@@ -24,6 +25,13 @@ class TicketType(FromToModel):
     def __str__(self):
         return self.title
 
+    def status(self):
```

Comment:
add [@property](https://docs.python.org/3/library/functions.html#property) decorator so we can use status as model property... (see comment below for more info)